backend/routes/courses.js

```diff
@@ -20,6 +20,48 @@ router.get("/getCourseByTag", async function(req, res, next) {
   }
 });
 
+/*
+* @api {post} /createCourse
+* @apiDescription This endpoint will add a course and an associated user
```

Comment:
```suggestion
* @apiDescription This endpoint will add a course
```
Unless I misunderstood your feature, I believe you are only adding a course without associating any user.